visualization/app/codeCharta/util/customConfigHelper.ts

```diff
@@ -166,26 +161,18 @@ export class CustomConfigHelper {
 		let count = 0
 
 		for (const config of CustomConfigHelper.customConfigs.values()) {
-			if (config.assignedMaps.join(" ") === mapNames && config.mapSelectionMode === mapSelectionMode) {
+			const configMapNames = [...config.assignedMaps.values()]
+			if (configMapNames.join(" ") === mapNames && config.mapSelectionMode === mapSelectionMode) {
 				count++
 			}
 		}
 
 		return count
 	}
 
-	static getConfigNameSuggestionByFileState(customConfigFileStateConnector: CustomConfigFileStateConnector): string {
-		const suggestedConfigName = customConfigFileStateConnector.getJointMapName()
-
-		if (!suggestedConfigName) {
```

Comment:
I assume this early return made no sense anyway?

Reply:
Yes. There should never be no files selected. That is why we do not need a early return here.